README.md

```diff
@@ -266,6 +266,19 @@ application, you should __always__ benchmark if performance matters to you.
 Please visit [Fastify help](https://github.com/fastify/help) to view prior
 support issues and to ask new support questions.
 
+All previous versions of Fastify besides the latest are end of life and will not receive any security or bug fixes.
+
+Our partner, HeroDevs, provides commercial security fixes for all unsupported versions at https://herodevs.com/fastify-nes.
+
+| Version | Supported? | Commercial Support         |
+| :------ | :--------- | :------------------------: |
+| 5       | Yes        |                            |
+| 4       | Yes        |                            |
+| 3       | No         | [Available here][hd-link]  |
+| 2       | No         | [Available here][hd-link]  |
+| 1       | No         | [Available here][hd-link]  |
```

Comment:
Do we really need to replicate the LTS doc table here?

Reply:
Using the user's hat: I prefer a table over a text wall

Reply:
I'm just not convinced it needs to be in two places. I think we could link to the LTS doc in the main readme, and expand the table there.

Reply:
I agree.

Reply:
Hey all, we can definitely change the table to just some simple text. 

We have operated with other projects on the assumption that users prefer simple tables with clear information without needing to click links to get full information, so we suggest this little table on all the projects we work with.

If you would prefer just text that's no problem. Please confirm which you want.

Reply:
Removing....